files/pull-sandbox-image.sh

```diff
@@ -8,4 +8,9 @@ if [[ "$(sudo ctr --namespace k8s.io image ls | grep $sandbox_image)" != "" ]];
   exit 0
 fi
 
-/etc/eks/containerd/pull-image.sh "${sandbox_image}"
+ecr_password=$(aws ecr get-login-password)
```

Comment:
if there is a problem with getting the password, will the timer (in the other PR) fail and then fail to re-fire again in the next time period?

Reply:
The timer will trigger the unit whether it failed or succeeded on previous runs, per my testing.

I copied over the `retry` logic we have in the other `pull-image.sh` script anyway.